### PR TITLE
[CR] Lower max stat cap in character creation

### PR DESCRIPTION
**Pull request by anothersimulacrum**
#### Summary
SUMMARY: Balance "Lower max stat cap in character creation to 14"

#### Purpose of change
There's really no reason for the ability to start with more than top-tier athlete stats. Mutants might have higher stats, but that should be done with stat boosting mutations, not with stats taken in character generation.

#### Describe the solution
Lower the cap to 14, and the point at which costs double to 12.

#### Describe alternatives you've considered
A different point at which costs double? I'm not sure what that should be, that's why this is CR.

#### Testing
Make a new character, attempt to increase any stat past 14.

### CODE REVIEW

File: src/game_constants.h [hunk 1]
```diff
@@ -118,7 +118,7 @@ constexpr time_duration CORPSE_ROT_TIME = 24_hours;
 #define MAX_SKILL 10
 
 // Maximum (effective) level for a stat.
-#define MAX_STAT 20
+#define MAX_STAT 14
```
Comment (LaVeyanFiend) [inline src/game_constants.h]: Is this intended to lower max cap on _all stats_, or just strength per the PR title?

Comment (anothersimulacrum) [reply src/game_constants.h]: All stats, whoops! Little bit of tunnel vision there.

**Comment by I-am-Erk** [comment]
This is simple enough that I feel comfortable merging, even if it is a c++ change... And I think now, while the early game is as easy as it's ever been, is a good time.

**Comment by Night-Pryanik** [comment]
The PR has a [CR] tag, but it was merged so quickly that I just didn't have a time to say a word about it. "Great" discussion we had here, what else can I say.

**Comment by anothersimulacrum** [comment]
I was mainly requesting comments on where the point on stat cost doubling should begin. If you have anything to say there, please do and I can possibly make another PR.

**Comment by Night-Pryanik** [comment]
I don't have objections on point of stat cost doubling, but I do have objection on the very idea of lowering the max stats cap.

**Comment by I-am-Erk** [comment]
A large number of people looked at it and agreed with it, and the change is pretty mundane. The stat cap part isn't even terribly negotiable, it's been the intent to drop it to 14 for a really long time. I think Kevin had more harsh plans.

**Comment by Night-Pryanik** [comment]
> A large number of people looked at it and agreed with it

The simple amount of people who liked the change NEVER was the reason for merging the PR. We had lots of PRs with literally tons of dislikes that was merged nevertheless.

> The stat cap part isn't even terribly negotiable

Yes, I know you're the boss with merge rights, nothing to discuss here.

**Comment by I-am-Erk** [comment]
I'm not talking about the thumbs up, I'm talking about the dev team. The thumbs up was a convenient way of knowing that people had seen the PR, but that wasn't a major factor in merging it

**Comment by Tamiore** [comment]
On a sidenote, I also like this change because the lower the cap for starting strength is, the less of an advantage you get from min/maxing every single point into it. 
It does not directly fix strength being the best stat, but at least it's somewhat alleviates the impact of maximizing it.

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/profession-affects-stat-maximums-on-chargen-lowering-chargen-max-stats/22902/5

**Pull request merged**